libs/designsystem/src/lib/components/modal/alert/alert.component.spec.ts

```diff
@@ -96,9 +105,46 @@ describe('AlertComponent', () => {
     it('should render', () => {
       component.iconName = 'warning';
       fixture.detectChanges();
-
       const icon = fixture.debugElement.query(By.css('.icon-outline'));
+
       expect(icon).toBeDefined();
     });
   });
 });
+
+describe('AlertComponent with okBtn', () => {
+  let spectator: SpectatorHost<AlertComponent>;
+  let element: HTMLElement;
+
+  const createHost = createHostFactory({
+    component: AlertComponent,
+    declarations: [ButtonComponent],
+    providers: [
+      {
+        provide: WindowRef,
+        useValue: window,
+      },
+    ],
+  });
+
+  beforeEach(() => {
+    spectator = createHost(`
+    <kirby-alert okBtnText="OK Button Text">
+    </kirby-alert>
+    `);
+    element = spectator.element;
+  });
+
+  it('should create', () => {
+    expect(spectator.component).toBeTruthy();
+  });
+
+  it('should have success colors on button', () => {
+    const okButton = element.getElementsByClassName('ok-btn')[0];
```

Comment:
Suggestion: You could use `querySelector()` or `querySelectorAll()` to get the button element.

```
const okButton = element.querySelector('.ok-btn');
```

or

```
const okButton = element.querySelectorAll('.ok-btn')[0];
```